Framework/DataHandling/test/LoadPSIMuonBinTest.h

```diff
@@ -69,21 +69,22 @@ class LoadPSIMuonBinTest : public CxxTest::TestSuite {
     TS_ASSERT(ws);
 
     TS_ASSERT_EQUALS(ws->getTitle(), "BNFSO      - Run:1529");
-    TS_ASSERT_EQUALS(ws->getLog("Field")->value(), "0.000G");
+    TS_ASSERT_EQUALS(ws->getLog("sample_magn_field")->value(), "0");
     TS_ASSERT_EQUALS(
         ws->getComment(),
         "Ba3NbFe3Si2O14, crystal                                       ");
-    TS_ASSERT_EQUALS(ws->getLog("Actual Temperature1")->value(), "4.999610");
-    TS_ASSERT_EQUALS(ws->getLog("Actual Temperature2")->value(), "5.197690");
+    TS_ASSERT_DELTA(std::stod(ws->getLog("Spectra 1 Temperature")->value()),
+                    4.99961, 0.00001)
+    TS_ASSERT_DELTA(std::stod(ws->getLog("Spectra 2 Temperature")->value()),
+                    5.19769, 0.00001)
     TS_ASSERT_EQUALS(ws->getLog("end_time")->value(), "2011-07-04T11:56:24");
     TS_ASSERT_EQUALS(ws->getLog("start_time")->value(), "2011-07-04T10:40:23");
-    TS_ASSERT_EQUALS(ws->getLog("Spectra0 label and scalar")->value(),
-                     "Forw - 14493858");
-    TS_ASSERT_EQUALS(ws->getLog("Spectra3 label and scalar")->value(),
-                     "Rite - 38247601");
-    TS_ASSERT_EQUALS(ws->getLog("Length of Run")->value(),
-                     "10.000000MicroSeconds");
-    TS_ASSERT_EQUALS(ws->getLog("Set Temperature")->value(), "5.000K");
+    TS_ASSERT_EQUALS(ws->getLog("Label Spectra 0")->value(), "Forw");
+    TS_ASSERT_EQUALS(ws->getLog("Scalar Spectra 0")->value(), "14493858");
+    TS_ASSERT_EQUALS(ws->getLog("Label Spectra 3")->value(), "Rite");
+    TS_ASSERT_EQUALS(ws->getLog("Scalar Spectra 3")->value(), "38247601");
+    TS_ASSERT_EQUALS(ws->getLog("Length of Run")->value(), "10");
+    TS_ASSERT_EQUALS(ws->getLog("sample_temp")->value(), "5");
```

Comment:
this should be a number not a string

Reply:
The value in the sample log is a numeric however when the value is returned using value() which I think is the only way, or at least the only way I have seen, it's returned as a string, I could do the conversion but I don't think that is necessary in it's current state.